estate/models/__init__.py

```diff
@@ -0,0 +1,2 @@
+
+from . import estate_property
```

Comment:
You are missing an empty end line at the end of some of the file of your PR. You should configure your IDE to add it automatically. In Github, it is visible with the little red circle at the bottom of the file. Also, you don't need an empty line at the first line. Same comment applies for the other init file